app/jobs/scheduled/poll_feed.rb

```diff
@@ -101,7 +101,9 @@ def url
       end
 
       def content
-        @article_rss_item.content.try(:force_encoding, "UTF-8").try(:scrub) || @article_rss_item.description.try(:force_encoding, "UTF-8").try(:scrub)
+        @article_rss_item.content_encoded&.force_encoding("UTF-8")&.scrub ||
+          @article_rss_item.content&.force_encoding("UTF-8")&.scrub ||
+          @article_rss_item.description&.force_encoding("UTF-8")&.scrub
```

Comment:
I haven't found much references for the `content` tag. Most RSS feeds that I've been fiddling with seems to be using the [`content:encoded` tag](http://www.rssboard.org/rss-profile#namespace-elements-content-encoded), namely the [Dota2 Blog Feed](http://blog.dota2.com/feed/), [Discourse Blog Feed](https://blog.discourse.org/feed), etc.

Do you think it'll be safe to remove the parsing for the `content` tag, and let whoever using it to adapt to our change? (Should be a minority)